docs/tools/MASTG-TOOL-0105.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Corellium
+platform: generic
+source: https://corellium.com
+---
+
+Corellium is an iOS and Android device virtualization platform that allows users to create and manage virtual devices, perform dynamic analysis, and test applications in a controlled environment.
+
+## Overview
+
+Corellium offers a cloud-based solution that enables users to run virtualized iOS and Android devices. These virtual devices can be used for various purposes, including security testing, app development, and research. Corellium provides a web-based interface for managing the virtual devices, as well as APIs for automation and integration with other tools.
+
+## iOS emulation
+
+Corellium is the only available commercial option for iOS emulation. It is possible to launch all types of iOS devices with any supported iOS version. Each device can be jailbroken from the start, so even recent versions of iOS can be used to analyze applications.
+
+Through the GUI, Corellium provides multiple features that are interesting for security testing:
+
+* Built-in file browser
+* Built-in Frida server
+* App overview and IPA installer
+* Certificate-pinning bypass (may not always work)
+* Snapshot management
+
+While Corellium has some very powerfull tools to analyze both applications and iOS itself, it does have a few important limitations:
+
+* **No App Store**: The devices do not have the App Store, which means you cannot use a Corellium device to obtain a decrypted version of an IPA file.
+* **No Apple Services**: Access to Apple services (including iMessage and push notifications) is unavailable.
+* **No Camera / Cellular / NFC / Bluetooth**: Apps running on Corellium do not have access to these peripherals.
```

Comment:
https://mas.owasp.org/contributing/5_Style_Guide/#write-so-people-will-read-with-joy

> Use dashes - rather than asterisks * for lists

We have a linting rule for it (MD004), not sure why it's not triggered: https://github.com/OWASP/owasp-mastg/blob/master/.markdownlint.jsonc#L4 

```suggestion
- **No App Store**: The devices do not have the App Store, which means you cannot use a Corellium device to obtain a decrypted version of an IPA file.
- **No Apple Services**: Access to Apple services (including iMessage and push notifications) is unavailable.
- **No Camera / Cellular / NFC / Bluetooth**: Apps running on Corellium do not have access to these peripherals.
```